### ISSUE TITLE
`nhost dev` fails on MacBook M1

### ISSUE THREAD

**Issue by moklick** [nhost/cli#85]
Hey,

when I try to run `nhost dev` I got the following error:

```sh 
Error! Command failed: docker-compose -f ./.nhost/docker-compose.yaml up -d --build
Creating network "nhost_default" with the default driver
Building nhost-api
#1 [internal] load build definition from Dockerfile-api
#1 sha256:9dc71cc303aa5135a404d06ec9d2d02299b429faea66de84209bdb365008439b
#1 transferring dockerfile: 162B 0.0s done
#1 DONE 0.3s

#2 [internal] load .dockerignore
#2 sha256:f1b4dbcbaec387b681129d45e70afb413f6e690502e7fa0eaa07d71932d013e6
#2 transferring context: 2B done
#2 DONE 0.6s

#3 [internal] load metadata for docker.io/nhost/nodeapi:v0.2.6
#3 sha256:f676c9da183169d422c61dd15d06b5123805a5df9af7ffbfdc5826d436a5d0bc
#3 DONE 0.7s

#4 [1/4] FROM docker.io/nhost/nodeapi:v0.2.6@sha256:3b8e9fce5e61eff92a7e5cd19b8f5a2fb28e044adaa05b44da6ff7c61e69c899
#4 sha256:5dbb7c58875933a0e1ed988c5fec1266949bd7b67b518c2f28e24d1d164c5579
#4 DONE 0.0s

#6 [internal] load build context
#6 sha256:9847883123cefc111c1e4c56870b41093753e8440051804f0f2ee1c721a8d328
#6 transferring context: 9.50kB 0.0s done
#6 DONE 0.2s

#5 [2/4] WORKDIR /usr/src/app
#5 sha256:3aad981cec18f58edff37180d22aa1f39d45f6bde46c5a6df4d3f09a8a7969e2
#5 CACHED

#7 [3/4] COPY api ./api
#7 sha256:ddb68579eb1732488444c3685590514b4b54e252c90012cb00ce794c367d971a
#7 CACHED

#8 [4/4] RUN ./install.sh
#8 sha256:85a830f38f7e97771368e0e72fe1ff9d2b717eeac50f70277d62001c50304594
#8 0.688 install.sh: applet not found
#8 ERROR: executor failed running [/bin/sh -c ./install.sh]: exit code: 127
------
 > [4/4] RUN ./install.sh:
------
executor failed running [/bin/sh -c ./install.sh]: exit code: 127
Service 'nhost-api' failed to build
```

All dependencies are installed. 

`which docker` returns `/usr/local/bin/docker`
`which docker-compose` returns `/usr/local/bin/docker-compose`
`which hasura` returns `./node_modules/.bin/hasura`

**Comment by elitan** [comment]
Can you test the solution proposed here: https://github.com/nhost/cli/pull/77 ?

**Comment by moklick** [comment]
The error still occurs:

```sh
nhost dev

✖ Nhost is starting...first run takes longer
Error! Command failed: docker-compose -f ./.nhost/docker-compose.yaml up -d --build
Creating network "nhost_default" with the default driver
Pulling nhost-graphql-engine (fedormelexin/graphql-engine-arm64:1.3.3)...
Building nhost-api
#2 [internal] load .dockerignore
#2 sha256:4a0f02aab2de35bd9f59061f6ee28a3463b137babfafbb3236818c91f76aed24
#2 transferring context: 2B done
#2 ...

#1 [internal] load build definition from Dockerfile-api
#1 sha256:953cf39bee660f7833472a12bddf67d925f179fde631610ec42a1b136ce683ef
#1 transferring dockerfile: 162B 0.0s done
#1 DONE 0.3s

#2 [internal] load .dockerignore
#2 sha256:4a0f02aab2de35bd9f59061f6ee28a3463b137babfafbb3236818c91f76aed24
#2 DONE 0.5s

#3 [internal] load metadata for docker.io/nhost/nodeapi:v0.2.6
#3 sha256:f676c9da183169d422c61dd15d06b5123805a5df9af7ffbfdc5826d436a5d0bc
#3 DONE 1.6s

#4 [1/4] FROM docker.io/nhost/nodeapi:v0.2.6@sha256:3b8e9fce5e61eff92a7e5cd19b8f5a2fb28e044adaa05b44da6ff7c61e69c899
#4 sha256:5dbb7c58875933a0e1ed988c5fec1266949bd7b67b518c2f28e24d1d164c5579
#4 DONE 0.0s

#6 [internal] load build context
#6 sha256:4a646442832d90d6e5877ce9332ac49cb94f5ee24d3dcfaeadae07eab800f0e2
#6 transferring context: 9.50kB 0.0s done
#6 DONE 0.2s

#5 [2/4] WORKDIR /usr/src/app
#5 sha256:3aad981cec18f58edff37180d22aa1f39d45f6bde46c5a6df4d3f09a8a7969e2
#5 CACHED

#7 [3/4] COPY api ./api
#7 sha256:55af934a6f3bd06bb4dc6c56dc38460c17b8757e801390d9f97e5835ef3a99af
#7 CACHED

#8 [4/4] RUN ./install.sh
#8 sha256:a106c90a0d34cd8529d5b2d2a69593be40b15230d208d19ec80086456e6c4f06
#8 0.717 install.sh: applet not found
#8 ERROR: executor failed running [/bin/sh -c ./install.sh]: exit code: 127
------
 > [4/4] RUN ./install.sh:
------
executor failed running [/bin/sh -c ./install.sh]: exit code: 127
Service 'nhost-api' failed to build

Error! Command failed: docker-compose -f ./.nhost/docker-compose.yaml up -d --build
Creating network "nhost_default" with the default driver
Pulling nhost-graphql-engine (fedormelexin/graphql-engine-arm64:1.3.3)...
Building nhost-api
#2 [internal] load .dockerignore
#2 sha256:4a0f02aab2de35bd9f59061f6ee28a3463b137babfafbb3236818c91f76aed24
#2 transferring context: 2B done
#2 ...

#1 [internal] load build definition from Dockerfile-api
#1 sha256:953cf39bee660f7833472a12bddf67d925f179fde631610ec42a1b136ce683ef
#1 transferring dockerfile: 162B 0.0s done
#1 DONE 0.3s

#2 [internal] load .dockerignore
#2 sha256:4a0f02aab2de35bd9f59061f6ee28a3463b137babfafbb3236818c91f76aed24
#2 DONE 0.5s

#3 [internal] load metadata for docker.io/nhost/nodeapi:v0.2.6
#3 sha256:f676c9da183169d422c61dd15d06b5123805a5df9af7ffbfdc5826d436a5d0bc
#3 DONE 1.6s

#4 [1/4] FROM docker.io/nhost/nodeapi:v0.2.6@sha256:3b8e9fce5e61eff92a7e5cd19b8f5a2fb28e044adaa05b44da6ff7c61e69c899
#4 sha256:5dbb7c58875933a0e1ed988c5fec1266949bd7b67b518c2f28e24d1d164c5579
#4 DONE 0.0s

#6 [internal] load build context
#6 sha256:4a646442832d90d6e5877ce9332ac49cb94f5ee24d3dcfaeadae07eab800f0e2
#6 transferring context: 9.50kB 0.0s done
#6 DONE 0.2s

#5 [2/4] WORKDIR /usr/src/app
#5 sha256:3aad981cec18f58edff37180d22aa1f39d45f6bde46c5a6df4d3f09a8a7969e2
#5 CACHED

#7 [3/4] COPY api ./api
#7 sha256:55af934a6f3bd06bb4dc6c56dc38460c17b8757e801390d9f97e5835ef3a99af
#7 CACHED

#8 [4/4] RUN ./install.sh
#8 sha256:a106c90a0d34cd8529d5b2d2a69593be40b15230d208d19ec80086456e6c4f06
#8 0.717 install.sh: applet not found
#8 ERROR: executor failed running [/bin/sh -c ./install.sh]: exit code: 127
------
 > [4/4] RUN ./install.sh:
------
executor failed running [/bin/sh -c ./install.sh]: exit code: 127
Service 'nhost-api' failed to build


⠋ stopping Nhost
Writing logs to ./.nhost/nhost.log

✖ Failed to start docker-compose
```

**Comment by elitan** [comment]
I think it's related to this:

https://github.com/nhost/api-handler/issues/7

**Comment by moklick** [comment]
Ok, that sounds great! I will watch this issue. Thanks for the fast reply.

**Comment by elitan** [comment]
@moklick Can you try `npm install -g nhost@next` with this latest version and see if that works?

I don't have an M1 yet :( Waiting for the 16"! :D

**Comment by moklick** [comment]
oh wow it works :) This is really nice! Thanks!

**Comment by elitan** [comment]
Fixed in 353a59e5bd02494bd6d1020f4c34bc464ade0955
